config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
+  if Rails.env.development?
+      mount GraphiQL::Rails::Engine, at: '/v1/graphiql', graphql_path: 'graphql#execute'
```

Comment:
Originally had this as just"/graphql" but decided to add "v1" as I think that is convention.

Reply:
I remember hearing that graphql did not need the versioning convention since the routes do change over time